marrow/mailer/transport/ses.py

```diff
@@ -24,28 +24,36 @@ def __init__(self, config):
         
         self.region = config.pop('region', "us-east-1")
         config.pop('use') #boto throws an error if we leave this in the next line
+        config.pop('debug')
         self.config = config  # All other configuration directives are passed to connect_to_region.
         self.connection = None
     
     def startup(self):
-        self.connection = boto.ses.connect_to_region(self.region, **self.config)
+        self.connection = boto3.client('ses',region_name=self.region, **self.config)
     
     def deliver(self, message):
         try:
-            destinations = [r.encode(encoding='utf-8') for r in message.recipients]
-            response = self.connection.send_raw_email(str(message), message.author.encode(), destinations)
+            destinations = [str(r) for r in message.recipients]
+            response = self.connection.send_raw_email(
+                                    Destinations=destinations,
+                                    RawMessage={
+                                        'Data': str(message)
+                                    },
+                                    Source=str(message.author),
+                                )
+                                
```

Comment:
Can this not be represented more compactly? This much indentation, and line splitting for single values detracts, I feel.

```python
response = self.connection.send_raw_email(
        RawMessage = {'Data': str(message)},
        Source = str(message.author),
        Destinations = destinations,
    )
```

I'm thinking something like the above (adjusted to match the overall indentation level) would be near-optimal. I sketched it out as a two-liner, but that was pushing it too far. Note: keyword argument order does not matter, so I like to represent a "flow" of importance and data direction, thus {message, source → destination}. 😉